### PR TITLE
Stready state storage

### PR DESCRIPTION
**Pull request by rb004f**
This is a proposed model for storage in steady-state.  At the very least review

1. The conversion from density to pressure.  I am not convinced it is a 1-1 conversion. (see issue #234 for ref_storage)
2. Review the (reduced) variable space and constraints for modeling storage in steady-state, and the various relaxations.  It assumes the pressure-change unit is at the top of well, so the well resistance gets applied below the pressure change unit.  There is one flow (fs), a constant pressure at the well base a variable pressure between the pressure-change unit and the well pipe, and the standard variable pressure at the junction (the head of the well).  The pressure change unit can compress and decompress flow in either direction within bounds.
3. Check the well resistance calculation.

Once we are converged on the model, I will update the documentation.

### CODE REVIEW

File: src/core/data.jl [hunk 1]
```diff
@@ -1275,6 +1275,16 @@ function _calc_pipe_resistance(pipe::Dict{String,Any}, base_length, base_pressur
     return resistance
 end
 
+"Calculates the resistance of the well associated with as storage component"
+function _calc_well_resistance(storage::Dict{String,Any}, base_length)
+    lambda = storage["well_friction_factor"]
+    D = storage["well_diameter"]
+    L = storage["well_depth"] * base_length
+
+    resistance = (lambda * D) \ L
```
Comment (kaarthiksundar) [inline src/core/data.jl:1284]: This should be lambda * L / D

Comment (rb004f) [reply src/core/data.jl:1284]: The steady-state model is in the variable space of pressure (not density), so doesn't that make it D / L (see calc_pipe_resistance)?

**Review by kaarthiksundar — CHANGES_REQUESTED** [review]
Can we have discussion on the well equations?

File: src/core/data.jl (hunk 1 above)
Comment (kaarthiksundar) [reply src/core/data.jl:1284]: I keep forgetting. But yes, in the pressure-mass flow space, the cross-sectional area terms missing.

**Pull request merged**